packages/api-explorer/src/components/DocSDKs/DocSDKs.tsx

```diff
@@ -63,16 +70,16 @@ export const DocSDKs: React.FC<LanguageSDKProps> = ({ api, method, type }) => {
   const languageKey = () => (method ? method.name : type!.name)
 
   return (
-    <Box pt="large">
-      <Heading as="h3" color="palette.charcoal700" mb="large">
+    <Box pt="large" pb="large">
+      <StyledHeading as="h2" fontWeight="semiBold" mb="medium">
```

Comment:
@jhardy - I wonder (mostly pondering to myself) if we should allow for `Heading` font-weight setting at some global level via the theme. Food for (future) thought.